forte/data/data_store.py

```diff
@@ -384,12 +396,13 @@ def _new_group(self, type_name: str, member_type: str) -> List:
         Args:
             type_name: The fully qualified type name of the new entry.
             member_type: Fully qualified name of its members.
+            tid: ``tid`` of the ``Group`` entry.
```

Comment:
This, and other occurrences of `tid`, should expalin that `tid` can be optional and will be auto-assigned.